source/admin_manual/list_of_events.rst

```diff
@@ -1223,10 +1223,18 @@ Fields present in all HTTP events.
 +---------------------+------------------------------------------------------+
 | bytes_out           | Amount of data written, in bytes.                    |
 +---------------------+------------------------------------------------------+
+| dest_host           | Destination host.                                    |
++---------------------+------------------------------------------------------+
+| dest_port           | Destination port.                                    |
++---------------------+------------------------------------------------------+
+| method              | HTTP verb used.                                      |
++---------------------+------------------------------------------------------+
 | redirects           | Number of redirects done while processing request.   |
 +---------------------+------------------------------------------------------+
 | status_code         | HTTP result status code (integer).                   |
 +---------------------+------------------------------------------------------+
+| target              | Request target.                                      |
```

Comment:
An example here would be good too. I think it's the `/path` including `/`? But I'm not sure..